.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "throneteki-json-data"]
 	path = throneteki-json-data
-	url = https://github.com/throneteki/throneteki-json-data
+	url = https://github.com/throneteki-playtesting/throneteki-json-data
```

Comment:
This change definitely should not be made on the main repo

Reply:
Reverted in 31fa026dc6b1e7e530b9fea7fef084789d887b57

Can mark as resolved.